### DISCUSSION TITLE
FluentBreadcrumb and ComponentBase

### DISCUSSION THREAD

**Discussion by Andras-Csanyi** [microsoft/fluentui-blazor#287] [Q&A] [answered]
Hey All,

We have the FluentBreadcrumb component and its attributes do not match with the template. See below.

What is the plan here? Shall the `FluentBreadcrumb` inherit from `FluentComponentBase` as other components do, or the template should be cleaned up? Or there is a third thing I haven't recognised yet.

I send my PR as it is now. Once the situation above is clarified and there is a decision I'm happy to make the changes.

[link](https://github.com/microsoft/fast-blazor/blob/main/src/Microsoft.Fast.Components.FluentUI/Components/Breadcrumb/FluentBreadcrumb.razor.cs)
```c#
using Microsoft.AspNetCore.Components;

namespace Microsoft.Fast.Components.FluentUI;

public partial class FluentBreadcrumb
{
    /// <summary>
    /// Gets or sets the content to be rendered inside the component.
    /// </summary>
    [Parameter]
    public RenderFragment? ChildContent { get; set; }
}
```

[link](https://github.com/microsoft/fast-blazor/blob/main/src/Microsoft.Fast.Components.FluentUI/Components/Breadcrumb/FluentBreadcrumb.razor)
```razor
@namespace Microsoft.Fast.Components.FluentUI
@inherits FluentComponentBase
<fluent-breadcrumb @ref=Element
                   class="@Class"
                   style="@Style"
                   @attributes="AdditionalAttributes">
    @ChildContent
</fluent-breadcrumb>
```

**Comment by vnbaaij** [comment, marked as answer]
The code should definitely follow the `.razor` file!

Stange that this one slipped through. I would expect it to generate an error message along the lines of 'base classes must be the same'.

Can you include the change in your test PR? Otherwise, I'll change it somewhere in the coming days.

**Reply by Andras-Csanyi** [reply]
Yes, sure, I can update the PR accordingly.
